index.html

```diff
@@ -750,6 +750,11 @@ <h3>
           Let |transport:RTCIceTransport| be the {{RTCIceTransport}} object associated with |candidatePair|.
         </p>
       </li>
+      <li>
+        <p>
+          [= list/Append =] |candidatePair| to {{RTCIceTransport/[[CandidatePairs]]}}.
+        </p>
+      </li>
```

Comment:
Github won't let me comment on the line directly above this section, which is:
> Let |candidatePair:RTCIceCandidatePair| be the candidate pair that was [= formed =].

While readers can hover over the `candidatePair` variable to learn its type, I think we need to spell out in prose what the type is now, since to a non-interactive reader `candidatePair` reads like an internal concept so far in this algorithm. We need to expound on its type before adding it to [[CandidatePairs]] since objects there need to be usable with the new match algorithm.

Something like:

> Let |candidatePair:RTCIceCandidatePair| be a new `RTCIceCandidate` dictionary representing the candidate pair that was [= formed =].

Moreover, we probably also need to clarify where the two instances of RTCIceCandidate interface objects are coming from. Are they also created here using `new`?